config/database.yml

```diff
@@ -17,18 +17,18 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: db
```

Comment:
Use the [docker service name](https://github.com/basileberjot/happy-cat/blob/c131ca512dcdd75c25814522610cf282bce61a49/docker-compose.yml#L5) instead of something like `localhost` to ensure the correct host address gets used (since the internal docker network uses a different ip address than localhost... this messed me up a lot initially too 😩)

[Reference](https://docs.docker.com/compose/rails/#connect-the-database)